### ISSUE TITLE
[Bug]: replace doesn't work on a table without any key

### ISSUE THREAD

**Issue by domingozhang** [matrixorigin/matrixone#10997]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior
drop table if exists t1;
create table t1(a int,b int,c int);
insert into t1 values(1,1,1),(2,2,2),(3,3,3);
replace t1 values(4,4,4);
replace t1 values(1,4,4);

select * from t1;

+------+------+------+
| a    | b    | c    |
+------+------+------+
|    1 |    1 |    1 |
|    2 |    2 |    2 |
|    3 |    3 |    3 |
+------+------+------+
3 rows in set (0.05 sec)

### Expected Behavior

In MySQL
mysql> select * from t1;
+------+------+------+
| a    | b    | c    |
+------+------+------+
|    1 |    1 |    1 |
|    2 |    2 |    2 |
|    3 |    3 |    3 |
|    4 |    4 |    4 |
|    1 |    4 |    4 |
+------+------+------+

**Comment by w-zr** [comment]
fixed in #11034

**Comment by domingozhang** [comment]
fixed
